Apps/Sandcastle/gallery/Imagery Layers Split.html

```diff
@@ -68,27 +68,41 @@
 var slider = document.getElementById('slider');
 viewer.scene.imagerySplitPosition = (slider.offsetLeft) / slider.parentElement.offsetWidth;
 
-var dragStartX = 0;
+var handler = new Cesium.ScreenSpaceEventHandler(slider);
 
-document.getElementById('slider').addEventListener('mousedown', mouseDown, false);
-window.addEventListener('mouseup', mouseUp, false);
+var bMoveActive = false;
 
-function mouseUp() {
-  window.removeEventListener('mousemove', sliderMove, true);
-}
+function move(movement){
+	if(!bMoveActive){
+		return;
+	}
 
-function mouseDown(e) {
-  var slider = document.getElementById('slider');
-  dragStartX = e.clientX - slider.offsetLeft;
-  window.addEventListener('mousemove', sliderMove, true);
+	var nMove = movement.endPosition.x ;//- movement.startPosition.x;
+	var splitPosition = (slider.offsetLeft + nMove) / slider.parentElement.offsetWidth;
+	slider.style.left = 100.0 * splitPosition + "%";
+	viewer.scene.imagerySplitPosition = splitPosition;
 }
 
-function sliderMove(e) {
-  var slider = document.getElementById('slider');
-  var splitPosition = (e.clientX - dragStartX) / slider.parentElement.offsetWidth;
-  slider.style.left = 100.0 * splitPosition + "%";
-  viewer.scene.imagerySplitPosition = splitPosition;
-}
+handler.setInputAction(function(movement) {
+	bMoveActive = true;
+}, Cesium.ScreenSpaceEventType.LEFT_DOWN);
+handler.setInputAction(function(movement) {
+	bMoveActive = true;
+}, Cesium.ScreenSpaceEventType.PINCH_START);
+
+handler.setInputAction(function(movement) {
```

Comment:
This can be shoehorned to 
```
handler.setInputAction(move, Cesium.ScreenSpaceEventType.MOUSE_MOVE);
handler.setInputAction(move, Cesium.ScreenSpaceEventType.PINCH_MOVE);
```